helloworld/src/main/scala/com/salesforce/hw/OpBostonSimple.scala

```diff
@@ -0,0 +1,165 @@
+  * @param lstat   % lower status of the population
+  * @param medv    median value of owner-occupied homes in $1000's
+  */
+case class BostonHouse
+(
+  rowId: Int,
+  crim: Double,
+  zn: Double,
+  indus: Double,
+  chas: String,
+  nox: Double,
+  rm: Double,
+  age: Double,
+  dis: Double,
+  rad: Int,
+  tax: Double,
+  ptratio: Double,
+  b: Double,
+  lstat: Double,
+  medv: Double
+)
+
+/**
+ * A simplified TransmogrifAI example classification app using the Boston dataset
+ */
+object OpBostonSimple {
+
+  /**
+   * Run this from the command line with
+   * ./gradlew sparkSubmit -Dmain=com.salesforce.hw.OpBostonSimple -Dargs=/full/path/to/csv/file
+   */
+  def main(args: Array[String]): Unit = {
+    if (args.isEmpty) {
+      println("You need to pass in the CSV file path as an argument")
+      sys.exit(1)
+    }
+    val csvFilePath = args(0)
+    println(s"Using user-supplied CSV file path: $csvFilePath")
+
+    // Set up a SparkSession as normal
+    implicit val spark = SparkSession.builder.config(new SparkConf()).getOrCreate()
+    import spark.implicits._ // Needed for Encoders for the BostonHouse case class
+
+    ////////////////////////////////////////////////////////////////////////////////
+    // RAW FEATURE DEFINITIONS
+    /////////////////////////////////////////////////////////////////////////////////
+
+    // Define features using the OP types based on the data
+    val rowId = FeatureBuilder.Integral[BostonHouse].extract(_.rowId.toIntegral).asPredictor
+    val crim = FeatureBuilder.RealNN[BostonHouse].extract(_.crim.toRealNN).asPredictor
+    val zn = FeatureBuilder.RealNN[BostonHouse].extract(_.zn.toRealNN).asPredictor
+    val indus = FeatureBuilder.RealNN[BostonHouse].extract(_.indus.toRealNN).asPredictor
+    val chas = FeatureBuilder.PickList[BostonHouse].extract(x => Option(x.chas).toPickList).asPredictor
+    val nox = FeatureBuilder.RealNN[BostonHouse].extract(_.nox.toRealNN).asPredictor
+    val rm = FeatureBuilder.RealNN[BostonHouse].extract(_.rm.toRealNN).asPredictor
+    val age = FeatureBuilder.RealNN[BostonHouse].extract(_.age.toRealNN).asPredictor
+    val dis = FeatureBuilder.RealNN[BostonHouse].extract(_.dis.toRealNN).asPredictor
+    val rad = FeatureBuilder.Integral[BostonHouse].extract(_.rad.toIntegral).asPredictor
+    val tax = FeatureBuilder.RealNN[BostonHouse].extract(_.tax.toRealNN).asPredictor
+    val ptratio = FeatureBuilder.RealNN[BostonHouse].extract(_.ptratio.toRealNN).asPredictor
+    val b = FeatureBuilder.RealNN[BostonHouse].extract(_.b.toRealNN).asPredictor
+    val lstat = FeatureBuilder.RealNN[BostonHouse].extract(_.lstat.toRealNN).asPredictor
+    val medv = FeatureBuilder.RealNN[BostonHouse].extract(_.medv.toRealNN).asResponse
+
+
+    // Define a feature of type vector containing all the predictors you'd like to use
+    val features = Seq(crim, zn, indus, chas, nox, rm, age, dis, rad, tax, ptratio, b, lstat).transmogrify()
+
+    val labels = medv
```

Comment:
Can you call it `label` so that people don't think it's a multi-label classification problem?